Makefile

```diff
@@ -62,6 +62,17 @@ changelog_check: ## check changelog format
 	./tools/generate-changelog.py --check legacy/intermediate_fw
 	./tools/generate-changelog.py --check python
 
+changelog_style: ## fix changelog format
+	./tools/generate-changelog.py --fix-style core
+	./tools/generate-changelog.py --fix-style core/embed/projects/boardloader
+	./tools/generate-changelog.py --fix-style core/embed/projects/bootloader
+	./tools/generate-changelog.py --fix-style core/embed/projects/bootloader_ci
+	./tools/generate-changelog.py --fix-style core/embed/projects/prodtest
+	./tools/generate-changelog.py --fix-style legacy/bootloader
+	./tools/generate-changelog.py --fix-style legacy/firmware
+	./tools/generate-changelog.py --fix-style legacy/intermediate_fw
+	./tools/generate-changelog.py --fix-style python
```

Comment:
this is becoming rather fragile, instead embed this list into `generate-changelog`
and make it fail if someone tries to use a non-listed path, telling them to also add the thing they want to the known path list

(that way we have a single command for "changelog check" and "changelog style" and we enforce that every changelog used by someone will be added to that list)

Reply:
Sounds good, thanks for the suggestion!
WDYT about 01325265a?

Reply:
(I've also added @mmilata, since he is the author of `tools/generate-changelog.py`)